cluster.go

```diff
@@ -633,12 +644,24 @@ func (c *Cluster) ID(ctx context.Context) *api.ID {
 		peers, _ = c.consensus.Peers(ctx)
 	}
 
+	clusterPeerInfos := c.peerManager.PeerInfos(peers)
```

Comment:
This change block is because the PeerManager does not return `api.Multiaddr` type anymore (which is our serializable wrapper for multiaddresses). Instead we need to build that address list by hand.